tmva/tmva/test/DNN/Utility.h

```diff
@@ -488,6 +488,32 @@ auto maximumRelativeError(const Matrix1 &X, const Matrix2 &Y) -> Double_t
    return maxError;
 }
 
+/** Compute the average element-wise absolute error of the matrices
+ *  X and Y.
+ */
+
+//______________________________________________________________________________
+template <typename Matrix1, typename Matrix2>
```

Comment:
Is it needed to template the type of both matrices? Shouldn't they be of the same type?

Reply:
I see that the implementation of the `maximumRelativeError` has done the some... Nevertheless, is it needed?

Reply:
Yeah, I guess we need the template arguments. Because in certain cases, the matrices can be of different types like TMatrixT or TCpuMatrix or TCudaMatrix.

Reply:
alright, fine then.